src/main/java/org/example/bookstoremate/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package org.example.bookstoremate.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "books")
+@Data
```

Comment:
please read about problem of using Data on JPA entities
Use @Getter/@Setter instead for now